cabal.project

```diff
@@ -0,0 +1,13 @@
+with-compiler: ghc-8.6.3
```

Comment:
Why does this file include a particular compiler version? What happens if users do not have that version?

Reply:
if I commented on the `with-compiler` it will use GHC PATH.